Makefrag

```diff
@@ -19,8 +19,9 @@ JVM_MEMORY ?= 2G
 SBT ?= java -Xmx$(JVM_MEMORY) -Xss8M -XX:MaxPermSize=256M -jar $(base_dir)/sbt-launch.jar
 SHELL := /bin/bash
 
+ROCKET_CLASSES := $(base_dir)/target/scala-2.11/classes
```

Comment:
This is b/c the black box source transform needs to find the stuff packaged in `src/main/resources`